plugins/modules/jenkins_plugin.py

```diff
@@ -821,6 +844,7 @@ def main():
         url_password=dict(no_log=True),
         version=dict(),
         with_dependencies=dict(default=True, type='bool'),
+        force_basic_auth=dict(default=False, type='bool')  # Add force_basic_auth to arguments
```

Comment:
For backwards compatibility the default needs to be `True`. Also please remove the comment, and add a trailing comma:
```suggestion
        force_basic_auth=dict(default=True, type='bool'),
```
You also need to add this default to the documentation in `DOCUMENTATION`, something like:
```yaml
options:
  [...]
  force_basic_auth:
    default: true
```